automl/cloud-client/get_operation_status_test.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+from google.cloud import automl
+import pytest
+
+import get_operation_status
+
+PROJECT_ID = os.environ["AUTOML_PROJECT_ID"]
+
+
+@pytest.fixture(scope="function")
+def get_operation_id():
+    client = automl.AutoMlClient()
+    project_location = client.location_path(PROJECT_ID, "us-central1")
+    response = client.transport._operations_client.list_operations(
+        project_location, ""
+    )
+    operation_id = ""
+    for operation in response:
```

Comment:
I don't follow this `for` loop. Is the goal to just get the first operation? Any reason you can't use something like:
```python
operation_id = response[0].name
```

Reply:
I'll put a a note, but if you try to access the iterator directly you get this error:
```
TypeError: 'GRPCIterator' object is not subscriptable
```
And you can't use `next()` as it throws this error:
```
TypeError: 'GRPCIterator' object is not an iterator
```
Then I didn't see a way to access one item directly.

Reply:
Does `response.pages.next()` work? 

Reference: [page_iterator.pages](https://googleapis.dev/python/google-api-core/latest/page_iterator.html#google.api_core.page_iterator.GRPCIterator.pages), [page_iterator.next](https://googleapis.dev/python/google-api-core/latest/page_iterator.html#google.api_core.page_iterator.Page.next)

Reply:
`response.pages` throws:
```
ValueError: ('Iterator has already started', <google.api_core.page_iterator.GRPCIterator object at 0x1117f2610>)
```

But I can get it to work by calling `pages` on the request first so that it doesn't start the iterator.
```
generator = client.transport._operations_client.list_operations(project_location, "").pages
page = next(generator)
operation = page.next()
yield operation.name
```

Reply:
Hmm okay, that makes 0 sense to me but that makes me less unhappy than the for loop.